mode-lambda/backend/gl.rkt

```diff
@@ -87,7 +87,10 @@
       ;; xxx allow these to be updated
       (define SpriteAtlasId (make-2dtexture))
       (with-texture (GL_TEXTURE0 SpriteAtlasId)
-        (load-texture/bytes atlas-size atlas-size atlas-bs))
+        (load-texture/bytes atlas-size atlas-size atlas-bs)
+        (glGenerateMipmap GL_TEXTURE_2D)
```

Comment:
I am happy making this optional, but if you want to, then add a parameter like `gl-filter-mode` (Maybe `gl-smoothing?`) and add it to the call to `define-make-delayed-render` next to `gl-filter-mode` then add an extra argument to `make-draw` after `screen-mode` and just add a `(when smooth? ....)` around these three lines.

Reply:
I will attempt to do that.